src/presentational/Classroom.jsx

```diff
@@ -16,7 +38,14 @@ export default class Classroom extends Component {
             <Tab>Groups</Tab>
           </TabList>
           <TabPanel>
-            Overview
+            <h3>Join Link</h3>
+            <p>Send the following URL to all the students you want to join this classroom.</p>
+            <p>Note: students need to login to Zooniverse.org to be able to join.</p>
+            <input className="form-control" type="text" value={this.state.url} readOnly/>
+            <CopyToClipboard text={this.state.url} onCopy={this.onCopy}>
+              <button className="btn btn-default">Copy to clipboard</button>
+            </CopyToClipboard>
+            {this.state.copied ? <span style={{color: 'red'}}>Copied :)</span> : null}
```

Comment:
Observation: clicking the Copy To Clipboard button causes the URL to be saved to be clipboard correctly (woot!) but the red-coloured 'Copied :)' text doesn't appear. This does not break functionality, but may be worth looking to in a future update.